content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,337 @@
+
+{{< language-toggle >}}
+
+{{< code shell "YML" >}}
+sensuctl dump core/v2.CheckConfig \
+--namespace dev \
+--format yaml | > sensu/namespaces/dev/checks.yml
+{{< /code >}}
+
+{{< code shell "JSON" >}}
+sensuctl dump core/v2.CheckConfig \
+--namespace dev \
+--format wrapped-json | > sensu/namespaces/dev/checks.json
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+Repeat this command for each resource type in each of your namespaces.
+
+##### Strip namespaces from resource definitions
+
+To [replicate and reuse resources][5] in any namespace without manual editing, create a copy of your existing resources with the namespaces stripped from their definitions:
+
+{{< language-toggle >}}
+
+{{< code shell "YML" >}}
+sensuctl dump all \
+--all-namespaces \
+--format yaml | grep -v "^\s*namespace:" > sensu/resources.yml
+{{< /code >}}
+
+{{< code shell "JSON" >}}
+sensuctl dump all \
+--all-namespaces \
+--format wrapped-json | grep -v "^\s*namespace:" > sensu/resources.json
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+## Implement monitoring as code with SensuFlow
+
+There's no one "correct" way to implement monitoring as code, but the [SensuFlow GitHub Action][1] offers a turnkey reference implementation that helps you create your own monitoring as code workflow and start managing Sensu resources via repository commits.
+
+SensuFlow is a git-based, prescriptive monitoring as code workflow that uses [sensuctl][2] (including [sensuctl prune][3]) to synchronize your monitoring and observability code with your Sensu deployments.
+
+{{% notice note %}}
+**NOTE**: SensuFlow is available for technical preview, and individual components in the workflow may change.
+Before you use SensuFlow in production, test it in a development environment or a dedicated test namespace in your current environment.
+{{% /notice %}}
+
+SensuFlow requires:
+
+- A monitoring code repository of Sensu resource definitions
+- A Sensu [role-based access control (RBAC)][4] service account with permission to manage all resources in your repository
+- A resource labeling convention to designate which resources the SensuFlow workflow should manage
+- Integration with your CI/CD system to run sensuctl commands as the service account user from the repository of resource definitions
+
+Read the [SensuFlow GitHub Action marketplace page][1] and [Monitoring as code with Sensu Go and SensuFlow][12] for more information.
+
+## Use CI/CD with monitoring as code
+
+When you're ready to expand your monitoring as code practices to involve CI/CD and multiple Sensu clusters, you can bootstrap your Sensu clusters with service accounts that are granted permissions for managing Sensu configuration.
+
+For example, the following role and role binding configuriations allow subjects to manage all configuration in a given namespace:
+
+TODO: add role and role binding examples
```

Comment:
Would this be the same thing I started trying to explain in https://github.com/sensu/sensu-docs/pull/2935/files?

Reply:
This section needs a bit more explanation about how CI/CD would benefit a monitoring as code approach.